lightning/src/routing/scoring.rs

```diff
@@ -1273,13 +1264,12 @@ impl<G: Deref<Target = NetworkGraph<L>>, L: Deref, T: Time> ScoreLookUp for Prob
 			_ => {},
 		}
 
-		let amount_msat = usage.amount_msat;
+		let amount_msat = usage.amount_msat.saturating_add(usage.inflight_htlc_msat);
```

Comment:
Won't this affect calculations for parameters like `base_penalty_amount_multiplier_msat`, `liquidity_penalty_amount_multiplier_msat`, etc? Those help avoid fees dominating the channel cost, but those fees will be calculated using `amount_msat`.

Reply:
Here, cost is penalty plus fees, so I think what I said was accurate, no?

Anyhow, we'll need to update the configuration docs now that this includes in-flight HTLCs.

Reply:
There are a few places where we say "payment amount" and sometimes explicitly `amount_msat`, e.g.: https://github.com/lightningdevkit/rust-lightning/blob/a7e3575ccc7cba00f712df6cfe5dab6d9ca17cee/lightning/src/routing/scoring.rs#L457-L464